packages/outstatic/package.json

```diff
@@ -32,6 +32,8 @@
     "@tiptap/extension-placeholder": "2.0.0-beta.202",
     "@tiptap/react": "2.0.0-beta.202",
     "@tiptap/starter-kit": "2.0.0-beta.202",
+    "camelcase": "^6.3.0",
+    "capitalize": "^2.0.4",
```

Comment:
I'd suggest dropping these.

1. Display names should be stored as written by the user
2. Internal IDs (for conflict resolution) should collapse the name into a consisted format. For example `${name}`.toLowerCase().trim()

That would allow us to drop both camelcase and capitalize as dependencies. As an added bonus, we don't have the headache of localizing for either of those scenarios anymore.

Reply:
I've removed capitalize, but I think camelcase makes sense. It will standardise data fetching, for example:
'Website URL' becomes 'websiteUrl' instead of 'websiteurl'. Camelcase is basically used for the markdown tags.

Getting rid of an additional library would be cool, but I think in this case we make the naming convention for fetching data inconsistent by not doing so.

Thoughts?